lldb/test/API/commands/target/anon-struct/main.cpp

```diff
@@ -0,0 +1,19 @@
+struct A {
+  struct {
+    int x = 1;
+  };
+  int y = 2;
+} a;
+
+struct B {
+  // Anonymous struct inherits another struct.
+  struct : public A {
+    int z = 3;
+  };
+  int w = 4;
```

Comment:
Are `w` and `a` necessary for the reproducer?

Reply:
No; leftovers from previous testing. Will remove.